api/templates/registration/password_change_form.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block body %}
+
+<nav aria-label="breadcrumb">
```

Comment:
Don't we have the breadcrumb for "change password" in the base.html file? Or this doing something else?